torch/utils/_sympy/value_ranges.py

```diff
@@ -540,7 +540,7 @@ def where(a, b, c):
             return ValueRanges(sympy.Min(b.lower, c.lower), sympy.Max(b.upper, c.upper))
 
     def __getattr__(self, name):
-        log.warning("unhandled ValueRange op %s", name)
+        log.debug("unhandled ValueRange op %s", name)
```

Comment:
How come?

Reply:
So running the benchmark script for BertForMaskedLM will execute this line a lot complaining about random seed related attributes. Those looks not actionable so I suppress the warning here to avoid flooding the output.  Maybe we can change this log to warning again once the underlying problem is fixed.